vnext/ReactUWP/Modules/Animated/AnimationDriver.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "AnimationDriver.h"
+
+namespace react {
+  namespace uwp {
```

Comment:
[nit] From what I've seen we follow the style of keeping the namespace in the same line: namespace react { namespace uwp {